src/js/jquery.uniform.js

```diff
@@ -610,11 +610,15 @@
                 apply: function ($el, options) {
                     var ds, $div, $filename, $button;
 
+                    // Issue# 441: Check if the control supports multiple selection.
```

Comment:
Fix as "#441" please. If you can, squash your commits into a single commit and update your PR.